sources/packages/backend/apps/api/src/route-controllers/institution/_tests_/e2e/institution.aest.controller.searchInstitutions.e2e-spec.ts

```diff
@@ -0,0 +1,140 @@
+import { HttpStatus, INestApplication } from "@nestjs/common";
+import * as request from "supertest";
+import {
+  AESTGroups,
+  BEARER_AUTH_TYPE,
+  createTestingAppModule,
+  getAESTToken,
+} from "../../../../testHelpers";
+import {
+  E2EDataSources,
+  createE2EDataSources,
+  createFakeInstitution,
+} from "@sims/test-utils";
+import { SearchInstitutionAPIOutDTO } from "../../models/institution.dto";
+
+describe("InstitutionAESTController(e2e)-searchInstitutions", () => {
+  let app: INestApplication;
+  let db: E2EDataSources;
+  const FAKE_INSTITUTION_NAME = "Search E2E Test institution name";
+
+  beforeAll(async () => {
+    const { nestApplication, dataSource } = await createTestingAppModule();
+    app = nestApplication;
+    db = createE2EDataSources(dataSource);
+  });
+
+  it("Should return institution(s) when searched by legal name.", async () => {
+    // Arrange
+    const institution = createFakeInstitution();
+    institution.operatingName = FAKE_INSTITUTION_NAME;
+    // Using business guid to keep the legal operating name unique for institution.
+    institution.legalOperatingName = institution.businessGuid;
```

Comment:
We do not need to reuse the business GUID we can just generate a new one, right?

Reply:
We do not have to re-use but I did not get the advantage/benefit of using a new uuid unless I am missing something.
I did the change considering it is very small.